lib/bolt/outputter/human.rb

```diff
@@ -311,6 +311,12 @@ def print_module_list(module_list)
         end
       end
 
+      def print_targets(options)
+        targets = options[:targets].map(&:name)
+        @stream.puts targets.join("\n")
+        @stream.puts colorize(:green, "#{targets.count} targets")
```

Comment:
I don't think we care too much about this, but it'd be nice if we did '1 target' when there's exactly one target. I think '0 targets' reads better than '0 target', so I'd leave that be.